internal/service/ldapverify/resource_ldap_verify_test.go

```diff
@@ -14,48 +14,22 @@ import (
 )
 
 const (
-	resourceName = "mongodbatlas_ldap_verify.test"
+	resourceName   = "mongodbatlas_ldap_verify.test"
+	dataSourceName = "data.mongodbatlas_ldap_verify.test"
 )
 
 func TestAccLDAPVerify_basic(t *testing.T) {
-	var (
-		orgID       = os.Getenv("MONGODB_ATLAS_ORG_ID")
-		hostname    = os.Getenv("MONGODB_ATLAS_LDAP_HOSTNAME")
-		username    = os.Getenv("MONGODB_ATLAS_LDAP_USERNAME")
-		password    = os.Getenv("MONGODB_ATLAS_LDAP_PASSWORD")
-		port        = os.Getenv("MONGODB_ATLAS_LDAP_PORT")
-		projectName = acc.RandomProjectName()
-		clusterName = acc.RandomClusterName()
-	)
-
-	resource.Test(t, resource.TestCase{
-		PreCheck:                 func() { acc.PreCheckLDAP(t) },
-		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
-		Steps: []resource.TestStep{
-			{
-				Config: configBasic(projectName, orgID, clusterName, hostname, username, password, cast.ToInt(port)),
-				Check: resource.ComposeTestCheckFunc(
-					checkExists(resourceName),
-					resource.TestCheckResourceAttrSet(resourceName, "project_id"),
-					resource.TestCheckResourceAttrSet(resourceName, "request_id"),
-					resource.TestCheckResourceAttr(resourceName, "hostname", hostname),
-					resource.TestCheckResourceAttr(resourceName, "bind_username", username),
-					resource.TestCheckResourceAttr(resourceName, "port", port),
-				),
-			},
-		},
-	})
+	resource.ParallelTest(t, *basicTestCase(t))
```

Comment:
```suggestion
	resource.Test(t, *basicTestCase(t)) // creating multiple ldap_verify resources for the same project at the same time leads to 500 errors.
```
defining basic tests (in ldap_verify and ldap_configuration) to run in parallel was not intentional right? In addition for the case of the migration tests I believe it is also running in parallel (not a significant issue currently as there is only 1 migration test).
As a good practice we can leave a small comment clarifying why we do not run in parallel, for the case of ldap_verify I was able to confirm that when running in parallel 1 return 500 error code during create.

Reply:
you're right, it's the only parallel test here so it's not a different in this case but it's better to be explicit, thanks